2017-09-13 23:31:25,058 INFO: microblog startup [in D:\microblog\app\__init__.py:40]
2017-09-13 23:34:54,007 ERROR: Exception on /edit [POST] [in D:\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\base.py", line 1182, in _execute_context
    context)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\default.py", line 470, in do_execute
    cursor.execute(statement, parameters)
sqlite3.IntegrityError: UNIQUE constraint failed: user.nickname

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "D:\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "D:\microblog\app\views.py", line 106, in edit
    db.session.commit()
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\scoping.py", line 157, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\session.py", line 906, in commit
    self.transaction.commit()
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\session.py", line 461, in commit
    self._prepare_impl()
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\session.py", line 441, in _prepare_impl
    self.session.flush()
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\session.py", line 2171, in flush
    self._flush(objects)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\session.py", line 2291, in _flush
    transaction.rollback(_capture_exception=True)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\util\langhelpers.py", line 66, in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\util\compat.py", line 187, in reraise
    raise value
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\session.py", line 2255, in _flush
    flush_context.execute()
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\unitofwork.py", line 389, in execute
    rec.execute(self)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\unitofwork.py", line 548, in execute
    uow
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\persistence.py", line 177, in save_obj
    mapper, table, update)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\orm\persistence.py", line 737, in _emit_update_statements
    execute(statement, multiparams)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\base.py", line 945, in execute
    return meth(self, multiparams, params)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\sql\elements.py", line 263, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\base.py", line 1053, in _execute_clauseelement
    compiled_sql, distilled_params
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\base.py", line 1189, in _execute_context
    context)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\base.py", line 1402, in _handle_dbapi_exception
    exc_info
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\util\compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\util\compat.py", line 186, in reraise
    raise value.with_traceback(tb)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\base.py", line 1182, in _execute_context
    context)
  File "D:\microblog\flask\lib\site-packages\sqlalchemy\engine\default.py", line 470, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) UNIQUE constraint failed: user.nickname [SQL: 'UPDATE user SET nickname=?, about_me=? WHERE user.id = ?'] [parameters: ('Jian', 'haha....', 2)]
2017-09-19 10:43:35,361 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:43:36,619 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:51:28,582 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:51:30,851 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:51:33,148 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:51:35,507 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:52:04,461 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:52:05,573 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:57:13,622 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 10:57:14,722 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 14:41:58,521 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 15:44:28,992 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 15:44:34,658 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 16:15:22,005 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 16:46:23,508 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 16:46:37,897 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-19 16:48:05,462 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-21 14:07:39,023 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-21 14:08:30,227 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-09-25 11:25:03,477 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-09 08:53:31,445 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-11 17:20:52,295 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-11 17:21:01,593 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-11 17:21:02,835 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 09:24:10,200 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 09:24:11,145 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:14:46,529 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:15:02,616 ERROR: Exception on /index [GET] [in C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\wang\Desktop\microblog\app\views.py", line 39, in index
    title='ึ๗าณ'
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\wang\Desktop\microblog\app\templates\index.html", line 13, in template
    {% endfor %}
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 896, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2017-10-16 13:18:57,272 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:19:05,320 ERROR: Exception on /index [GET] [in C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\wang\Desktop\microblog\app\views.py", line 39, in index
    title='ึ๗าณ'
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\wang\Desktop\microblog\app\templates\index.html", line 13, in template
    {% endfor %}
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 896, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2017-10-16 13:19:20,422 ERROR: Exception on /index [GET] [in C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\wang\Desktop\microblog\app\views.py", line 39, in index
    title='ึ๗าณ'
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\wang\Desktop\microblog\app\templates\index.html", line 13, in template
    {% endfor %}
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 896, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2017-10-16 13:19:22,222 ERROR: Exception on /index [GET] [in C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\wang\Desktop\microblog\app\views.py", line 39, in index
    title='ึ๗าณ'
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\wang\Desktop\microblog\app\templates\index.html", line 13, in template
    {% endfor %}
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 896, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2017-10-16 13:19:24,034 ERROR: Exception on /index [GET] [in C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\wang\Desktop\microblog\app\views.py", line 39, in index
    title='ึ๗าณ'
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\wang\Desktop\microblog\app\templates\index.html", line 13, in template
    {% endfor %}
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 896, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2017-10-16 13:19:25,370 ERROR: Exception on /index [GET] [in C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\wang\Desktop\microblog\app\views.py", line 39, in index
    title='ึ๗าณ'
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\wang\Desktop\microblog\app\templates\index.html", line 13, in template
    {% endfor %}
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 896, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2017-10-16 13:23:56,772 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:23:58,413 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:24:02,935 ERROR: Exception on /index [GET] [in C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask_login\utils.py", line 228, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\wang\Desktop\microblog\app\views.py", line 39, in index
    title='ึ๗าณ'
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\wang\Desktop\microblog\app\templates\index.html", line 13, in template
    {% endfor %}
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 896, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 878, in subparse
    rv = self.parse_statement()
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\wang\Desktop\microblog\flask\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2017-10-16 13:28:48,994 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:28:50,724 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:57:59,312 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 13:58:00,488 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:01:57,905 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:02:05,349 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:02:06,495 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:09:46,452 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:09:47,612 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:14:35,091 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:14:36,252 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:28:18,405 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:28:19,514 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:40:22,018 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 14:43:37,075 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 15:42:04,874 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 16:04:11,896 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 16:05:25,127 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 16:05:26,514 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 16:23:00,520 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 16:39:04,681 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 16:39:05,875 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 16:40:10,257 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 17:06:46,330 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 17:20:08,626 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 17:25:17,820 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 17:25:18,948 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 17:27:30,693 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-16 17:33:41,248 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-17 08:54:23,868 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:40]
2017-10-17 08:56:58,414 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 08:59:28,878 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 09:00:01,603 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 09:00:36,069 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 09:07:45,097 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 10:57:27,325 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 10:59:29,129 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:01:39,839 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:02:35,528 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:03:22,467 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:04:19,610 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:05:10,515 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:05:24,939 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:06:28,045 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:07:42,445 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:08:25,093 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:09:21,803 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:10:54,972 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:16:57,395 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 11:17:32,814 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 13:31:54,847 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 14:31:11,633 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 14:31:12,799 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 14:35:26,229 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 14:35:38,642 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 14:35:39,833 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 14:37:17,195 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 15:49:38,521 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 16:53:21,034 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 16:53:25,348 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
2017-10-17 17:17:44,036 INFO: microblog startup [in C:\Users\wang\Desktop\microblog\app\__init__.py:42]
